cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -84,6 +84,373 @@ constexpr inline auto is_supported_construction_value_type()
 // Helper functions for `fixed_point` type
 namespace detail {
 
+/**
+ * @brief Helper struct containing large compile-time-computed powers of 10
+ */
+struct PowersOf10 {
```

Comment:
suggestion: we use `snake_case` for class, type and variable names, other than template parameters.